general-superstaq/general_superstaq/user_config.py

```diff
@@ -165,21 +168,22 @@ def aqt_download_configs(
         variables_file_path: Optional[str] = None,
         overwrite: bool = False,
     ) -> Optional[Tuple[Dict[str, Any], Dict[str, Any]]]:
-        """Downloads AQT configs that had previously been uploaded to SuperstaQ, optionally saving
-        them to disk. Reading AQT configurations requires the PyYAML package.
+        """Downloads AQT configs that had previously been uploaded to SuperstaQ.
+
+        Optionally saves configs to disk. Reading AQT configurations requires the PyYAML package.
```

Comment:
nit (to make a little more clear how the files are saved, and that PyYAML is only required when no filenames are passed):
```suggestion
        Optionally saves configs to disk as YAML configuration files. Otherwise, the PyYAML package is required to read the downloaded configs.
```